mne_bids/tsv_handler.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+from collections import OrderedDict, Counter
+
+
+def from_tsv(fname, dtypes=None):
+    """Read a tsv file into an OrderedDict
+
+    Parameters
+    ----------
+    fname : str
+        Path to the file being loaded.
+    dtypes : list
+        List of types to case the values loaded as. This is specified column by
+        column.
+        Defaults to None. In this case all the data is loaded as strings."""
+    data = np.loadtxt(fname, dtype=str, delimiter='\t')
+    column_names = data[0, :]
+    info = data[1:, :]
+    data_dict = OrderedDict()
+    if isinstance(dtypes, (list, tuple)):
+        if len(dtypes) == info.shape[1]:
+            for i, name in enumerate(column_names):
+                data_dict[name] = info[:, i].astype(dtypes[i]).tolist()
+    else:
```

Comment:
why do you need this else? shouldn't `astype(dtypes[i])` work for all cases?

Reply:
not if you aren't trying to cast the data to a type...
`dtypes` defaults to `None`, so we'd have to get it to default to a list of `str`'s with the same length as the number of columns (which wouldn't actually be super hard, I just didn't think of it...)